dev/criteo/test-with-maven.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+# You can run tests for a module only by specifying -pl <module>
+# as argument to this script, or some specific test by passing
+# -Dtest=... (see surefire documentation).
+
+source `dirname "$0"`/criteo-spark-env.sh
+
+export _JAVA_OPTIONS="-Xss2048k -Dspark.buffer.pageSize=1048576 -Xmx4g"
```

Comment:
Do we really need those parameters 4g seems quite a lot.
By the way those settings should be set for maven: export MAVEN_OPTS="-Xmx2g -XX:MaxPermSize=512M -XX:ReservedCodeCacheSize=512m"